Specta/Specta/SpectaDSL.h

```diff
@@ -48,6 +48,15 @@ OBJC_EXTERN void sharedExamples(NSString *name, void (^block)(NSDictionary *data
 #define      itBehavesLike(...) spt_itShouldBehaveLike_(@(__FILE__), __LINE__, __VA_ARGS__)
 
 OBJC_EXTERN void waitUntil(void (^block)(DoneCallback done));
+/**
+ * Runs the @c block and waits until the @c done block is called or the
```

Comment:
is `@c` special here, or are you just using it as a contraction of completion ( which is called done ) ?